markdown/h2m/handlers/cards.ts

```diff
@@ -1,13 +1,35 @@
+import * as Gettext from "node-gettext";
+import * as fs from "fs";
+import * as path from "path";
+import { DEFAULT_LOCALE } from "../../../libs/constants";
 import { h } from "../h";
 import { asArray } from "../utils";
 import { toText } from "./to-text";
 import { QueryAndTransform } from "./utils";
 
+const gettextLocalizationMap = (function initLocalizationsMap() {
+  const getTextDefaultDomainName = "messages";
+  let gtLocalizationMap = new Map();
+  let localesOnFS = fs
+    .readdirSync(path.join(__dirname, "../../localizations"))
+    .map((str) => str.split(".")[0]);
+  localesOnFS.forEach((localeStr) => {
+    const translations = require("../../localizations/" + localeStr + ".json");
+    const gt = new Gettext();
+    gt.addTranslations(localeStr, getTextDefaultDomainName, translations);
+    gt.setLocale(localeStr);
+    gtLocalizationMap.set(localeStr, gt);
+  });
+  return gtLocalizationMap;
+})();
+
 export const cards: QueryAndTransform[] = [
   ...["note", "warning"].map(
     (className) =>
       [
-        (node) => {
+        (node, opts) => {
+          const locale = opts.locale || DEFAULT_LOCALE;
+          const gt = gettextLocalizationMap.get(locale);
```

Comment:
We should default this to English.

Reply:
I'm not sure if I get your intent right but please see 5239908

Reply:
@SphinxKnight 
Sorry that I didn't make myself clear, it's about `yarn md h2m web/html --locale de` not failing. We should default to the en-US keywords if there's on localization.

I'd even argue that the en-US keywords should always work even if it has been localized.

Reply:
agreed, I should have seen this one.
Thanks for the review, I've pushed cd26305 and hope it's OK.